setup/smartos/ansible-playbook.yaml

```diff
@@ -0,0 +1,77 @@
+---
+- hosts: iojs-build-smartos
+
+  remote_user: root
+
+  vars:
+    - ansible_python_interpreter: "/usr/bin/env python"
```

Comment:
It didn't work without it.  Although, in my PR it was `ansible_python_interpreter=/opt/local/bin/python`